src/main/java/sparksoniq/jsoniq/compiler/translator/expr/flowr/FlworExpression.java

```diff
@@ -77,6 +77,12 @@ private void set_returnClause(ReturnClause returnClause) {
     }
 
     @Override
+    public ExecutionMode getHighestExecutionMode() {
+        this._highestExecutionMode = ExecutionMode.UNSET;
```

Comment:
A side effect in a get function raises red flags. Is the default not unset anyway (because of init), and do we not just ignore it?

Reply:
You're absolutely right. That was simply an oversight on my part. That line was meant for the init method.

Reply:
Thanks!